tests/Objects/CorrectMultipleErrorTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+use CodingAvenue\Proof\Code;
+use Proofs\Proof;
+class CorrectMultipleErrorTest extends Proof
+{
+    public function testPhpStartTag()
+    {
+        $checkStart = self::$code->codeStartCheck();
+
+        $this->assertEquals(true, $checkStart, "Expecting the `<?php` tag on the first line.");
+    }
+
+    public function testActualCode()
+    {
+        $evaluator = self::$code->evaluator();
+        $evaled    = $evaluator->evaluate();
+        $expected  = "Charles";
+
+        $this->assertEquals($expected, $evaled['output'], "Expected output is \"$expected\".");
+    }
+
+    public function testEcho()
+    {
+        $nodes = self::$code->find('construct[name="echo"]');
+
+        $this->assertEquals(2, $nodes->count(), "Expecting two echo statements.");
+    }
+
+    public function testAssignment()
+    {
+        $nodes = self::$code->find('operator[name="assignment"]');
+
+        $this->assertEquals(2, $nodes->count(), "Expecting two assignment statements.");
+    }
+
+    public function testPersonVariable()
+    {
+        $person = self::$code->find('variable[name="person"]');
+
+        $this->assertEquals(3, $person->count(), "Expecting three occurrences of the variable named 'person'.");
+    }
+
+    public function testInstantiation()
+    {
+        $nodes = self::$code->find('instantiate[class="Person"]');
+
+        $this->assertEquals(1, $nodes->count(), "Expecting an instantiation statement of the 'Person' class.");
+    } 
+
+    public function testEat()
+    {
+        $eat = self::$code->find('method[name="eat"]');
+
+        $this->assertEquals(1, $eat->count(), "Expecting an eat() method.");
+    }
```

Comment:
@cerilacanda Sorry, I might have missed this in the previous PR. But this might have some potential problem. Check this code below
```
class Person {
}

class Person2 {
   public function eat() {
   }
}
```

The test for `eat` method will still be true, but we're expecting the `eat` method to be on the `Person` class.

Reply:
I think what we need to do something like this:
```
$obj = self::$code->find('class[name="Person"]');
$subNodes = $obj->getSubnode();
$eat = $subNodes->find('method[name="eat"]');
```

Reply:
The goal is to make sure that the `eat` method is part of the the `Person` class